devtools/index.js

```diff
@@ -49,7 +49,10 @@ let storeonDevtools = options => {
         window.top.__REDUX_DEVTOOLS_EXTENSION__
     } catch {}
     if (!extension) {
-      if (process.env.NODE_ENV !== 'production') {
+      if (
+        process.env.NODE_ENV !== 'production' &&
+        process.env.NODE_ENV !== 'test'
+      ) {
```

Comment:
Let me know if you prefer this condition instead.
```suggestion
      if (process.env.NODE_ENV === 'development') {
```